src/lib/server/editors/editors.ts

```diff
@@ -1,21 +1,20 @@
-import { ChainOfThoughtEditor } from './ChainOfThoughtEditor';
 import { LongerEditor } from './LongerEditor';
 import { NoNegationEditor } from './NoNegationEditor';
 import { OutputDescriptionEditor } from './OutputDescriptionEditor';
-import { OutputLabelsEditor } from './OutputLabelsEditor';
 import { SeparateInstructionEditor } from './SeparateInstructionEditor';
 import { ShortenEditor } from './ShortenEditor';
+import { SingleInstructionEditor } from './SingleInstructionEditor';
+import { JSONDescriptionEditor } from './json/JSONDescriptionEditor';
 import { JSONInstructionEditor } from './json/JSONInstructionEditor';
-import { JSONDescriptionEditor } from './json/JsonDescriptionEditor';
 
 export const editors = [
-	new ChainOfThoughtEditor(),
 	new NoNegationEditor(),
 	new SeparateInstructionEditor(),
 	new ShortenEditor(),
 	new LongerEditor(),
-	new OutputLabelsEditor(),
+	// new OutputLabelsEditor(),
```

Comment:
Do we want to collect deleted ones somewhere in case we want to revive them? Otherwise I'd just delete it.

Reply:
Can always go back to old commits.